<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraLumiere</class>
 <widget class="QDialog" name="CameraLumiere">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1403</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(&quot;C:/projet-new/projet/background1.jpg&quot;);background-size:cover;
background-repeat:no-repeat;</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1331</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Lumiere</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="idlum_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>idlum :</string>
      </property>
     </widget>
     <widget class="QLabel" name="etatlum_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Etat lumiere</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="duree">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="duree_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Duree</string>
      </property>
     </widget>
     <widget class="QLabel" name="piece_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Piece</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="piece">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="idlum">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QComboBox" name="etatlum">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ouvert</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>fermer</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QLineEdit" name="leRecherchelumiere">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>10</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>recherche</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAjouterlumiere">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSupprimerlumiere">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QTableView" name="tablumiere">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>791</width>
       <height>321</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonModifererlumiere">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbTrilumiere">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>10</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Tous</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>idlum</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>etatlum</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>duree</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>piece</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="DevoirVideo">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Video</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_date_time">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>360</y>
       <width>331</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="DvExl">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>370</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Excel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Camera</string>
    </attribute>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>1391</width>
       <height>531</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="afficherCarteFid">
      <widget class="QTableView" name="tabCarteFid">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>741</width>
         <height>361</height>
        </rect>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_recherche_carteFid">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>40</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>recherche</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_colone_carteFid">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>40</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Tous</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NUM_CARTE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DATE_DE_CREATION</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>POINTS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TOTAL_ACHATS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>STATUT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>REMISE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MATRICULE_FISCALE</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="button_ajouter_carteFid">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_supprimer_carteFid">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>470</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>470</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Retour</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="ajoutClient_2">
      <widget class="QDialogButtonBox" name="validerAjout_cf">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>500</y>
         <width>193</width>
         <height>28</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="modifierClient_2">
      <widget class="QDialogButtonBox" name="validerModif_cf">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>500</y>
         <width>193</width>
         <height>28</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
      <widget class="QComboBox" name="cbTricamera">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>10</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Tous</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>idcam</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>etatcam</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>emplacement</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>331</width>
         <height>351</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="NumLoisir">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>idcam:</string>
        </property>
       </widget>
       <widget class="QLabel" name="TypeLoisir">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>Etat Camera :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="emplacementcam">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>160</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="DateLoisir">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>Emplacement Cam :</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="idcam">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>50</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QComboBox" name="etatcam">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>100</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>ouvert</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>fermer</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QPushButton" name="buttonModiferercamera">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>370</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonSupprimercamera">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>370</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leRecherchecamera">
       <property name="geometry">
        <rect>
         <x>1010</x>
         <y>10</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>recherche</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonAjoutercamera">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>370</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QTableView" name="tabcamera">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>40</y>
         <width>891</width>
         <height>321</height>
        </rect>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="Pdfcamera">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>370</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Pdf</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Imprimercamera">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>370</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Imp</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="TradEngLois">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>425</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>English</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="TradFrLois">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>425</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Français</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="TradGerLois">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>425</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Deutsch</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LoisXl">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>370</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Excel</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
